06-building-a-program-start-to-finish/writing-tests/output/app-spec.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable */
+/**
+ * Dependencies declared globally
+ */
+const InvertedIndex = require('../app/js/app.js'),
```

Comment:
Is **_app.js_** your InvertedIndex? Then I believe the name of the file should reflect what the file is.